examples/web/async-large.html

```diff
@@ -85,9 +80,9 @@
         };p[i].q=p[i].q||[];n=l.createElement(o);g=l.getElementsByTagName(o)[0];n.async=1;
         n.src=w;g.parentNode.insertBefore(n,g)}}(window,document,"script","../sp.js","snowplow_2"));
 
-        window.snowplow_2('newTracker', 'cf', '<<MY_COLLECTOR_URL>>', { // Initialise a tracker
+        window.snowplow_2('newTracker', 'sp', '<<MY_COLLECTOR_URL>>', { // Initialise a tracker
```

Comment:
I think tracker namespace needs to be different. `sp2` perhaps? Maybe also change the collector var to illustrate going to two separate places.